src/ArcCommander/APIs/ConfigurationAPI.fs

```diff
@@ -173,4 +173,47 @@ module ConfigurationAPI =
         | false,false | false,true  -> 
             let workDir = GeneralConfiguration.getWorkDirectory arcConfiguration
             IniData.getLocalConfigPath workDir
-            |> unsetValueInIniPath
\ No newline at end of file
+            |> unsetValueInIniPath
+
+
+    /// Transfer git user metadata from arc config to git config.
+    let setGitUser (arcConfiguration : ArcConfiguration) (configurationArgs : Map<string,Argument>) =
+
+        let log = Logging.createLogger "ConfigurationSetGitUserLog"
+
+        log.Info("Start Config SetGitUser")
+
+        let nameOption = 
+            match tryGetFieldValueByName "Name" configurationArgs with
+            | Some name -> 
+                log.Trace("TRACE: Retrieved user name from given argument.")
+                Some name
+            | None -> 
+                log.Trace("TRACE: Could not retrieve user name from argument. Try to retrieve it from arc config.")
+                GeneralConfiguration.tryGetGitName arcConfiguration
+            
+        let emailOption = 
+            match tryGetFieldValueByName "Email" configurationArgs with
+            | Some name -> 
+                log.Trace("TRACE: Retrieved user email from given argument.")
+                Some name
+            | None -> 
+                log.Trace("TRACE: Could not retrieve user email from argument. Try to retrieve it from arc config.")
+                GeneralConfiguration.tryGetGitEmail arcConfiguration  
+
+        match nameOption, emailOption with
+        | Some name, Some email ->
+
+            if containsFlag "Global" configurationArgs then 
+                GitHelper.setGlobalName name
+                GitHelper.setGlobalEmail email
+                
+            if (containsFlag "Global" configurationArgs |> not) || containsFlag "Local" configurationArgs then
+                let workDir = GeneralConfiguration.getWorkDirectory arcConfiguration
+                GitHelper.setLocalName workDir name
+                GitHelper.setLocalEmail workDir email
+
+            log.Info("Finished setting git user information.")
+
+        | _, _ -> 
+            log.Error("ERROR: Git user metadata neither present in the arc config nor given by argument. Consider first running \"arc auth\", setting the user name and email by running \"arc config set\" or rerunning this command and specifying both name and email.")
```

Comment:
"[...] present in the ***ARC*** config [...]". "arc" only when referring to commands: "`arc auth`" etc.